### ISSUE TITLE
Add support for firefox

### ISSUE THREAD

**Issue by Robert-Beier** [Robert-Beier/jitsi-speaker-stats-exporter#6]
The extension was built for chrome, uses Chrome specific APIs, and is currently also only running on Chrome. But with some modifications, it could as well run on firefox. The `manifest.json` could maybe even stay the same. But the usage of chrome APIs needs a rework.

When adding firefox support, it'd also be great to create automated build processes for Chrome and Firefox from the same code base. This could be done for example with separating functionality with Filenames like `contentScript.chrome.js` and contentScript.firefox.js`.

**Comment by Gatienzo** [comment]
I hope you or someone else can develop a Firefox extension based on this one for Chrome. I am a teacher and need to know the list of participants. 
I searched but could not find any.
Also, I don´t know how to install in Chrome from here.
Thanks.

**Comment by Robert-Beier** [comment]
Good point. I just added the link to the extension in the chrome web store to the [README file](https://github.com/Robert-Beier/jitsi-speaker-stats-exporter). You can download it [here](https://chrome.google.com/webstore/detail/jitsi-speaker-stats-expor/mnokejfahilneembnhjdnpcdjgdklgff). It's actually only in german right now. I'll check how to translate it and how to port it to firefox.

**Comment by Gatienzo** [comment]
Thanks it worked!
There is not much to translate. It´s intuitive. Google translate worked well for Spanish.
I hope you can port it to Firefox.
